.gitignore

```diff
@@ -14,3 +14,6 @@ vetur.config.js
 beamer/data/relayers
 relayer/relayer-*-x64
 addresses.json
+0x1CEE82EEd89Bd5Be5bf2507a92a755dcF1D8e8dc.json
+agent-ganache-local.conf
+deployments/ganache-local
```

Comment:
I don't think we should add these: these are just examples from the docs, not something that is always there. Those paths could be named arbitrarily.